3_visualize/src/report-do-inventory.Rmd

```diff
@@ -0,0 +1,186 @@
+---
+title: "DRB dissolved oxygen data availability"
```

Comment:
It's just so pretty that it'd be a shame not to be able to see it easily!

Reply:
Sure, we could add and commit the rendered reports so that they're easily referenced as intermediate outputs/visual checks.

Reply:
Oooh. Can you try this? https://stackoverflow.com/a/39816334/5811195

GH wasn't rendering the HTML, but I think you can export to just `.md` using above and then it will render on GH.

Reply:
So this _seems_ like it should be trivial to do, but Rmarkdown makes working with subdirectories a pain. I tested this out a bit without much luck - even after exporting to .md the report isn't rendering correctly on GH.

I don't want to hold up this merge, so I suggest reverting to the USGS-R style of not committing /out files for now. I can open an issue to dig into this further depending on how we want to share visuals without necessarily building the pipeline locally. Sound OK, @jsadler2?

Reply:
@lekoenig I was messing around with this also,[ for another project ](https://github.com/galengorski/municipal-water-loss-ml/tree/main/1_data_inventory) and I was running into issues. I ended up putting a link to an rpub website in the repo and hosting the markdown file there. Not a great solution, but the markdown file is viewable without having to recompile.

Reply:
Thanks, @galengorski - it seems like it should be so easy! I messed around with a few options and wasn't satisfied with anything I was coming up with. I like your idea to host on an rpub website at least in the interim. I'll see if that can be integrated within the targets pipeline.

Reply:
> I don't want to hold up this merge, so I suggest reverting to the USGS-R style of not committing /out files for now.

Sounds good to me. I do like the idea of figuring out a solution for sharing results (that RPub is cool!) but seems like outside of this PR would be best.